portal/templates/redesign/teach_new/onboarding_school.html

```diff
@@ -71,7 +78,48 @@ <h4 class="col-sm-offset-2">Everything you need to start coding with your class
         </div>
     </div>
 </div>
+{% if teacher.pending_join_request %}
+<div id="onboarding_revoke_request_page"></div>
+
+<div class="sub-nav sub-nav--warning">
+    <div class="row">
+        <img src='{% static "portal/img_new/warning.png" %}'>
+        <p class="body-text"><i>Your request to join {{ teacher.pending_join_request.name }}, {{ teacher.pending_join_request.postcode }} has been submitted</i></p>
+    </div>
+</div>
+
+<div class="background col-sm-6 col-center">
+    <section>
+        <h2>Join a school</h2><br>
+    </section>
+    <h4>You have a pending request to join <b>{{ teacher.pending_join_request.name }}, {{ teacher.pending_join_request.postcode }}</b>.<br>Once the administrator of your school or club has approved your request you will be able
```

Comment:
No break.  And if you're going to use a break, then break your code line where you break your text.